infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -113,15 +114,16 @@ public ExecutorService getExecutor() {
      */
     public ManagementServer(ServerContext serverContext) {
 
-        this.opts = serverContext.getServerConfig();
-        this.localEndpoint = this.opts.get("--address") + ":" + this.opts.get("<port>");
+        managementServerConfig = ManagementServerConfig.parse(serverContext.getServerConfig());
+
+        localEndpoint = managementServerConfig.getAddress() + ":" +
+                             managementServerConfig.getPort();
         this.serverContext = serverContext;
 
         executor = Executors.newFixedThreadPool(serverContext.getManagementServerThreadCount(),
                 new ServerThreadFactory("management-", new ServerThreadFactory.ExceptionHandler()));
 
-        this.failureHandlerPolicy = serverContext.getFailureHandlerPolicy();
-        this.healingPolicy = serverContext.getHealingHandlerPolicy();
+        failureHandlerPolicy = serverContext.getFailureHandlerPolicy();
```

Comment:
Why was this changed?

Reply:
healingPolicy wasn't being used.
if you're referring to "this", I removed unnecessary keyword as per previous comments. similarly, if you feel strongly about the inclusion of this keyword, I'll bring them that. That being said there are other constructors in the code base where "this" only used for dealing with shadowed variables. let me know.

Reply:
I join to Vjeco and also vote for using `this` in a constructor,  just make it consistent and lined up the code. For any other methods don't use `this`

Reply:
done